stories/Modal.story.jsx

```diff
@@ -17,6 +19,91 @@ const LayoutStyles = styled.div`
 
 storiesOf("Modal", module)
   .add("basic", () => <Modal>A very basic modal.</Modal>)
+  .add(
+    "only Submit button",
+    () => {
+      const ModalUseCase = () => {
+        const [open, toggleModal] = useState(true);
+
+        return (
+          <>
+            <Button onClick={() => toggleModal(true)}>
+              Launch composed modal
+            </Button>
+            {open && (
+              <Modal
+                onClose={() => toggleModal(false)}
+                onSubmit={() => alert("I can submit something")}
+                submitText="Submit"
+              >
+                A very basic modal that wont scroll the background when opened.
+              </Modal>
+            )}
+          </>
+        );
+      };
+      return <ModalUseCase />;
+    },
+    {
+      info:
+        "Sumbit button is show since property 'onSubmit' is being passed on the component."
+    }
+  )
+  .add(
+    "only Cancel button",
+    () => {
+      const ModalUseCase = () => {
+        const [open, toggleModal] = useState(true);
+
+        return (
+          <>
+            <Button onClick={() => toggleModal(true)}>
+              Launch composed modal
+            </Button>
+            {open && (
+              <Modal
+                onClose={() => toggleModal(false)}
+                onCancel={() => alert("I can cancel something")}
+                cancelText="Cancel"
+              >
+                A very basic modal that wont scroll the background when opened.
+              </Modal>
+            )}
+          </>
+        );
+      };
+      return <ModalUseCase />;
+    },
+    {
+      info:
+        "Cancel button is show since property 'onCancel' is being passed on the component."
+    }
+  )
+  .add("both Cancel and Submit buttons", () => {
+    const ModalUseCase = () => {
+      const [open, toggleModal] = useState(true);
```

Comment:
whats the difference with the next story?

Reply:
the modal is not shown and you click the button to trigger it..

Reply:
doesn't think that necessary, since trigger btn is already here and you can click it

Reply:
@lanwen my bad. the last story would show that the modal wont actually scroll when you have it opened. Therefore I will add better description in the story so that everybody can tell what it is about but I think it makes sense not to mix the stories but keep it separate. 
What you think?